storage/ndb/rest-server/data-access-rondb/src/rdrs-hopsworks-dal.cpp

```diff
@@ -125,7 +125,7 @@ RS_Status find_api_key_int(Ndb *ndb_object, const char *prefix, HopsworksAPIKey
   }
 
   char cmp_str[API_KEY_PREFIX_SIZE];
-  memcpy(cmp_str + 1, prefix, API_KEY_PREFIX_SIZE - 1);
+  memcpy(cmp_str + 1, prefix, strlen(prefix) + 1); //+1 copy null terminator too
```

Comment:
Calling strlen twice is a bit of an overkill, please introduce a local variable to avoid that

Reply:
fixed